src/adaBase58.c

```diff
@@ -18,18 +18,20 @@
 #include "adaBase58.h"
 #include "assert.h"
 
+const uint8_t MAX_BUFFER_SIZE = 124;
```

Comment:
please, make it `static const` so that it isn't visible outside of this compilation unit. Also, I am a bit wary of `uint8_t` as a size type but let's deal with it later (I have a big todo on checking implicit integer conversions through the code)